pkg/kubectl/cmd/util/factory.go

```diff
@@ -554,10 +557,6 @@ func recordCommand(args []string) string {
 	return strings.Join(args, " ")
```

Comment:
per @deads2k's comment, this probably shouldn't do any cleanup, actually... if they ran this command with `cluster/kubectl.sh`, I would expect to see the actual command run, not some truncated version of it